libmscore/score.cpp

```diff
@@ -2295,11 +2295,19 @@ bool Score::appendScore(Score* score)
             return false;
       int tickOfAppend = lastMeasure->endTick();
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      if (addPageBreak) {
+            // add PageBreak if one doesn't already exist
+            if (!lastMeasure->pageBreak())
```

Comment:
Good point...I should modify this so that Page Break will replace preexisting Line Break.